LEGO1/lego/legoomni/src/actors/helicopter.cpp

```diff
@@ -455,7 +455,7 @@ void Helicopter::FUN_100042a0(const Matrix4& p_matrix)
 	Vector3 vec3(m_unk0x1a8[0]); // locala8  // esp+0x20
 	Vector3 vec4(m_unk0x1a8[1]); // localb8  // esp+0x10
 	Vector3 vec5(m_unk0x1a8[2]); // EDI
-	Vector3 vec6(m_unk0x1a8[3]); // locala0  // esp+0x28
+	Vector3 vec6((const float*)m_unk0x1a8[3]); // locala0  // esp+0x28
```

Comment:
Interesting - can you add a comment noting that this is required to make the function match?